openquakeplatform/openquakeplatform/svir/views.py

```diff
@@ -72,6 +72,48 @@
 @condition(etag_func=None)
 @allowed_methods(('GET', ))
 @sign_in_required
+def get_supplemental_information(request):
+    """
+    Given a layer name, read the corresponding supplemental_information from
+    the layer's metadata and return that as a json
```

Comment:
maybe this should read "Given a layer name, read the corresponding supplemental_information from the layer's metadata and return it as a json"